chatbot.py

```diff
@@ -1,12 +1,12 @@
 from utilities import parse_sentence
 from utilities import classify_model
 from utilities import classify_sentence
-from utilities import setup_database
-from utilities import add_to_database
-from utilities import get_chat_response
-from utilities import get_question_response
-from utilities import learn_question_response
-from utilities import add_learnt_statement_to_database
+from databaseconnect import setup_database
```

Comment:
Since we're essentially importing almost all (possibly all) functions from the `databaseconnect.py` module, I suggest we import the module and use that to refer to the functions. The same might apply to functions from other modules as well. This is a small issue and can be fixed within this PR. 
@vishakha-lall thoughts?

Reply:
@janakrajchadha that's a great suggestion, thanks for pointing that out. It would certainly make the code cleaner. However, since this is a 'beginner' level issue, I would recommend we create a new issue around this. Essentially, 'beginner' level issues require sticking to the description of the issue. Does that sound good?